packages/synthchron-tauri/src/components/react-flow/processModels/petriNet/customNodes/PlaceNode.tsx

```diff
@@ -13,85 +13,86 @@ const handleStyle = {
-        background: '#eee',
-        border: '1px solid #000000',
-        borderRadius: '50px',
-        textAlign: 'center',
-        color: 'red',
-        minWidth: config.diameter,
-        minHeight: config.diameter,
-      }}
-    >
-      <div className='handleContainer'>
-        <Handle
-          type='source'
-          position={Position.Top}
-          id='top'
-          isConnectable={isConnectable}
-          style={{
-            ...handleStyle,
-            top: '-15px',
-          }}
-        />
-        <Handle
-          type='source'
-          position={Position.Bottom}
-          id='bottom'
-          isConnectable={isConnectable}
-          style={{
-            ...handleStyle,
-            bottom: '-15px',
-          }}
-        />
-        <Handle
-          type='source'
-          position={Position.Right}
-          id='right'
-          isConnectable={isConnectable}
-          style={{
-            ...handleStyle,
-            right: '-15px',
-          }}
-        />
-        <Handle
-          type='source'
-          position={Position.Left}
-          id='left'
-          isConnectable={isConnectable}
-          style={{
-            ...handleStyle,
-            left: '-15px',
-          }}
-        />
-      </div>
+    <div style={{ position: 'relative' }}>
+      <Handle
+        id='bottom'
+        position={Position.Bottom}
+        type='source'
+        isConnectable={isConnectable}
+        style={{
+          ...handleStyle,
+          bottom: '-30px',
+        }}
+      />
+      <Handle
+        id='top'
+        position={Position.Top}
+        type='source'
+        isConnectable={isConnectable}
+        style={{
+          ...handleStyle,
+          top: '-30px',
+        }}
+      />
+      <Handle
+        id='right'
+        position={Position.Right}
+        type='source'
+        isConnectable={isConnectable}
+        style={{
+          ...handleStyle,
+          right: '-30px',
+        }}
+      />
+      <Handle
+        id='left'
+        position={Position.Left}
+        type='source'
+        isConnectable={isConnectable}
+        style={{
+          ...handleStyle,
+          left: '-30px',
+        }}
+      />
+      <svg
+        style={{ display: 'block', overflow: 'visible' }}
+        width={diameter}
+        height={diameter}
+      >
+        {shape}
+      </svg>
       <div
         style={{
           display: 'flex',
           justifyContent: 'center',
           alignItems: 'center',
           position: 'absolute',
-          top: -config.diameter / 2 - 25,
+          top: 0,
           left: 0,
           width: '100%',
           height: '100%',
           pointerEvents: 'none',
         }}
       >
-        <div
-          style={{
-            fontFamily: 'monospace',
-            fontWeight: 'bold',
-            color: '#222',
-            fontSize: 12,
-          }}
-        >
-          {data?.label}
-        </div>
+        {data?.storage}
```

Comment:
This line still references "storage". Incompatible with naming in other files

Reply:
Fixed it, sry.